### PR TITLE
ft-api-user-delete-email, add route to delete a specific message [finishes #164727466]

### PR DESCRIPTION
**Pull request by Ivankivu**
This PR adds the following route user function:

user delete email
test_delete_message

### CODE REVIEW

**Comment by jennizalwango** [comment]
l like your readme file and then logic of getting unread messages

**Comment by Ivankivu** [comment]
> l like your readme file and then logic of getting unread messages

Thank you! @jennizalwango. Did u find anything that might be in need of change or have a better alternative to what i do currently have?

File: README.md [hunk 1]
```diff
@@ -9,27 +15,31 @@ As EPIC Andelans who work towards advancing human potential and giving back to t
...
+ |- send email to individuals or groups||
 
-* As an Admin:
-    1. I should be able to create a group and add different users.
+ ## login details
+
+|user account|Admin account|
+|:---:|:---:|
+|email `ivan@gmail.com`||
+|password `brenda`||
```
Comment (mendozabree) [inline README.md]: Please include instructions on how the repo can be set up locally. Reach to either Jenny or Ahmed to help you.

Comment (Ivankivu) [reply README.md]: I am currently finishing up updating my readme to include all API information

File: app/api/views/views.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    def unread_message():
+        if not messages:
+            return jsonify({"error": "no new mail."}), 404
+        if ['status'] != 'unread':
+            return jsonify({"error": "no mail by  user found!"}), 404
+        return jsonify(messages)
+
+    @app.route('/api/v1/messages/<int:id>', methods=['GET', 'DELETE'])
+    def get_single_message_delete(id):
+            if not messages:
```
Comment (mendozabree) [inline app/api/views/views.py]: Lines 116 to 127 are not indented right. Please work on that.

Comment (Ivankivu) [reply app/api/views/views.py]: okay

File: README.md [hunk 3]
```diff
@@ -9,27 +15,31 @@ As EPIC Andelans who work towards advancing human potential and giving back to t
...
```
Comment (mendozabree) [inline README.md]: The `Installation` instructions can be better. 
Reach out to Jenny or Ahmed
Plus include the `Heroku` link

Comment (Ivankivu) [reply README.md]: I will have the changes in the next push to this PR to update the requested changes and Readme.

File: app/api/views/views.py (hunk 2 above)
Comment (Ivankivu) [reply app/api/views/views.py]: @mendozabree I have pushed the changes. kindly please find time to review

**Review by mendozabree — APPROVED** [review]
LGTM.

**Pull request merged**